docker/vpp-vswitch/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# compile vpp-agent needs vpp libs
+FROM contiv/vpp-base-builder:b59bd659 as builder
+
+ENV GOLANG_VERSION 1.9.3
+ENV GOPATH /root/go
+ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
+
+RUN apt-get update \
+    && apt-get install -y curl git make \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl -L "https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz" | tar -C /usr/local -xzf - \
+    && go get -u github.com/golang/lint/golint
+
+ADD . $GOPATH/src/github.com/contiv/vpp
```

Comment:
`COPY` is preferred over `ADD` unless you need the special stuff `ADD` can do:  https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy

Reply:
I thought ADD will do a compression+transport , looks like i misread the doc, will fix